truss/build.py

```diff
@@ -233,10 +233,12 @@ def init(
     """
     config = TrussConfig(
         model_type="custom",
```

Comment:
can we just drop this too and just construct with all the defaults with the exception of the python version?